### ISSUE TITLE
Cocoapods podspec for Swift 3 (SwiftLocation 1.1.0)

### ISSUE THREAD

**Issue by robbiet480** [malcommac/SwiftLocation#78]
Hi,

In the latest Podspec from the master branch there are two blocking issues:
1. `post_install` is not valid. Instead, you can set `s.pod_target_xcconfig = { 'SWIFT_VERSION' => '3.0' }`. I believe that this may be entirely unnecessary however. As it currently stands, Cocoapods outputs this error when trying to install from the master branch:

```
Fetching external sources
-> Pre-downloading: `SwiftLocation` from `https://github.com/malcommac/SwiftLocation.git`, branch `master`
  $ /usr/bin/git ls-remote https://github.com/malcommac/SwiftLocation.git master
  c85adba85e5a5c337a51d076739816ce5ee56b4f      refs/heads/master
 > Git download
 > Git download
     $ /usr/bin/git clone https://github.com/malcommac/SwiftLocation.git /var/folders/vb/p0k46yvx7c1gh61gt1_8k8_00000gn/T/d20160911-40099-19mintm --template=
     Cloning into '/var/folders/vb/p0k46yvx7c1gh61gt1_8k8_00000gn/T/d20160911-40099-19mintm'...
   $ /usr/bin/git checkout --quiet c85adba85e5a5c337a51d076739816ce5ee56b4f
[!] Unable to find a specification for 'SwiftLocation'.

/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/external_sources/abstract_external_source.rb:117:in `block in pre_download'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/user_interface.rb:85:in `titled_section'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/external_sources/abstract_external_source.rb:112:in `pre_download'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/external_sources/downloader_source.rb:13:in `fetch'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:609:in `fetch_external_source'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:585:in `block (2 levels) in fetch_external_sources'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:584:in `each'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:584:in `block in fetch_external_sources'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:583:in `fetch_external_sources'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer/analyzer.rb:73:in `analyze'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer.rb:236:in `analyze'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer.rb:150:in `block in resolve_dependencies'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer.rb:149:in `resolve_dependencies'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/installer.rb:110:in `install!'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/command/update.rb:60:in `run'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/lib/cocoapods/command.rb:50:in `run'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/gems/cocoapods-1.1.0.rc.1/bin/pod:55:in `<top (required)>'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/bin/pod:23:in `load'
/Users/robbiet480/.rvm/gems/ruby-2.2.2/bin/pod:23:in `<main>'

[!] Unable to load a podspec from `SwiftLocation.podspec`, skipping:

Pod::DSLError
```
1. The release tag referenced (`s.version`) is not yet published, so trying to install using `pod 'SwiftLocation', :git => 'https://github.com/malcommac/SwiftLocation.git', :branch => 'master'` fails (assuming you got past the `post_install` issue).

**Comment by robbiet480** [comment]
Using:

```
pod 'SwiftLocation', :git => 'https://github.com/malcommac/SwiftLocation.git', :commit => 'af2b78f3daf8e78c60537dd02103262676b53c3c'
```

allows the update/install to continue.

**Comment by malcommac** [comment]
Just fixed a podspec error; however there is a bug with cocoapods and swift 3 compiling.
See https://github.com/CocoaPods/CocoaPods/issues/5846
It should be fixed very soon, hope to post SwiftLocation 1.1.0 later today.

**Comment by malcommac** [comment]
Fixed. Pushed 1.1.0 for Swift 3.
